CONTRIBUTING.md

```diff
@@ -1,59 +1,91 @@
-
-## Pull Request Process
-
-1. Create a branch. For example, if you are going to work on issue number #44 (which is, say, a new feature for ‘forgot password’ management):
+# Contributing to Circuitverse
+
+- We expect contributors to abide by our underlying [Code of Conduct](https://github.com/CircuitVerse/CircuitVerse/blob/master/code-of-conduct.md) . All discussions about this project must be respectful and harassment-free.
+- Remember that communication is the lifeblood of any Open Source project. We are all working on this together, and we are all benefiting from this software.
+- It's very easy to misunderstand one another in asynchronous, text-based conversations. When in doubt, assume everyone has the best intentions.
+- If you feel anyone has violated our Code of Conduct, you should anonymously contact the team with our abuse report form via [Slack](https://circuitverse.org/slack), necessary action will be take by the team.
+
+## Issue label
+
+Please note if someone is already assigned or working on the issue, do not try to start working without asking in a thread.
+
+- `maintainers` label are internal tasks that will be completed by a Circuitverse core team member. 
+- [good first issue](https://github.com/CircuitVerse/CircuitVerse/labels/good%20first%20issue) labeled issues are meant for newer developers.
+- [feature](https://github.com/CircuitVerse/CircuitVerse/labels/%F0%9F%8C%9F%20feature) labeled issues are meant to propose new features.
+- [bugs](https://github.com/CircuitVerse/CircuitVerse/labels/%F0%9F%90%9E%20bug) labeled issues are meant to have errors in existing code base.
+- [domumentation](https://github.com/CircuitVerse/CircuitVerse/labels/documentation) labeled issues are meant to have typo errors in documentation.
+- [Simulator](https://github.com/CircuitVerse/CircuitVerse/labels/simulator) issues are meant specifically in the scope of circuitverse simulator.
+- [platform](https://github.com/CircuitVerse/CircuitVerse/labels/platform) labeled issues are meant specifically in the scope of circuitverse website.
+- `difficulty: easy` issues are usually confined to isolated areas of existing code.
+- `difficulty: medium` issues sometimes entail new features and might affect a significant area of the codebase, but aren't overly complex.
+- `difficulty: hard` issues are typically far-reaching, and might need architecture decisions during implementation. This label might also denote highly complex issues.
+- `duplicate` labeled issues are meant to be already existing issue in the repository.
+- `priority: less` labeled issues are meant to have priority comparatavily lesser than other issues.
+- `priority: medium` labeled issues are meant to have priority comparatively intermediate than other issues.
+- `priority: high` labeled issues are meant to have highest priority and need to fix as soon as possible.
```

Comment:
I think instead of  priority less, medium and hard we can add label p-1, p-2 and p-3 respectively. If an issue/pr has multiple labels by using short-terms we can reduce clutter 🤔.

Reply:
> no, not a new contributor may not be well equipped with short terms like p-1,p-2 etc.
We can add full forms in label description.